proxy/src/telemetry/metrics/labels.rs

```diff
@@ -115,12 +115,15 @@ impl RequestLabels {
 
 impl fmt::Display for RequestLabels {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self.authority {
-            Some(ref authority) =>
-                write!(f, "authority=\"{}\",{}", authority, self.direction),
-            None =>
-                write!(f, "authority=\"\",{}", self.direction),
-        }?;
+        if let Some(ref authority) = self.authority {
+            write!(f, "authority=\"{}\"", authority)?;
+        } else {
+            f.pad("authority=\"\"")?;
+        }
+
+        if let Some(ref direction) = self.direction {
```

Comment:
I think we may instead want to output a `direction="control"` label for control thread metrics? 

While "direction" may not really be the best label name for that, I believe it would mean that these events would show up in Grafana charts that are grouped by direction...